### PR TITLE
 Merge 1.10.x into master

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.mdx [hunk 1]
```diff
@@ -16,6 +16,20 @@ https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [1.10.15] - 2020-10-09
```
Comment (erohmensing) [inline CHANGELOG.mdx:19]: This should probably go under 2.0, but also we're missing the 1.13 and 1.14 changelogs 🤔

File: CHANGELOG.mdx [hunk 2]
```diff
@@ -16,6 +16,20 @@ https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
...
+### Bugfixes
+
+- [#3597](https://github.com/rasahq/rasa/issues/3597): Fixed issue where temporary model directories were not removed after pulling from a model server. If the model pulled from the server was invalid, this could lead to large amounts of local storage usage.
+- [#6755](https://github.com/rasahq/rasa/issues/6755): Treat the length of OOV token as 1 to fix token align issue when OOV occurred.
+- [#6899](https://github.com/rasahq/rasa/issues/6899): Fixed ``MappingPolicy`` not predicting ``action_listen`` after the mapped action while running ``rasa test``.
+
+
+### Improvements
+
+- [#6900](https://github.com/rasahq/rasa/issues/6900): Debug logs from ``matplotlib`` libraries are now hidden by default and are configurable with the ``LOG_LEVEL_LIBRARIES`` environment variable.
```
Comment (erohmensing) [inline CHANGELOG.mdx:30]: did the code for this make it into master?

Comment (tabergma) [reply CHANGELOG.mdx:30]: Seem like it. At least the change was already part of the Rasa 2.0 release according to our changelog.

File: CHANGELOG.mdx (hunk 1 above)
Comment (tabergma) [reply CHANGELOG.mdx:19]: the changes are not on master anymore, I guess we should can recreate them from our release notes

File: CHANGELOG.mdx (hunk 2 above)
Comment (erohmensing) [reply CHANGELOG.mdx:30]: So strange. I don't understand how any of this is working 😄

**Comment by tabergma** [comment]
@erohmensing There is a failing test. Is that related to the changes?

**Comment by erohmensing** [comment]
merge seems to have fixed it 🤔

File: CHANGELOG.mdx (hunk 1 above)
Comment (erohmensing) [reply CHANGELOG.mdx]: cool, thanks! yeah i see some got in here: https://github.com/RasaHQ/rasa/pull/6942/files

**Review by erohmensing — APPROVED** [review]

**Pull request merged**